src/Mod/Path/PathScripts/PathDeburrGui.py

```diff
@@ -135,7 +135,7 @@ def taskPanelBaseGeometryPage(self, obj, features):
 Command = PathOpGui.SetupOperation('Deburr',
         PathDeburr.Create,
         TaskPanelOpPage,
-        'Path-Deburr',
+        'Path_Deburr',
```

Comment:
I made the same assumption (as mentioned above) for all `PathOpGui.SetupOperation()` functions